_blog.yml

```
@@ -2102,3 +2102,11 @@
     - nlp
     - community
     - research
+
+- local: content-guidelines
```

```suggestion
- local: content-guidelines-update
```

maybe, to differentiate more (in search engines etc) from the actual /content-guidelines page